qemu/tests/cfg/timedrift_adjust_time.cfg

```diff
@@ -41,13 +48,16 @@
                 rtc_base = localtime
     variants:
         - guest_pause_resume:
-            only adjust_host_clock
+            only no_adjust_clock adjust_host_clock
             vm_action = pause_resume
             sleep_seconds = 1800
-            time_difference = 3600
+            time_difference = 1800
+            time_difference_hwclock = 0
         - guest_reboot:
+            no no_adjust_clock
```

Comment:
Could you explain this ?  It is a new change in think.

Reply:
Yes, new change, no need the case.

Reply:
I checked the test plan,  several cases reboot vm without adjust host and guest clock such as: RHEL7-6826, RHEL7-6827, RHEL7-6824, RHEL7-6825. Could you confirm it , thanks!

Reply:
Yes, they have been automated before. No need here.